modules/021-cni-cilium/rbac.yaml

```diff
@@ -0,0 +1,5 @@
+module: cni-cilium
+scopes:
+  - networking
+crds:
+  - modules/021-cni-cilium/crds/*.yaml
```

Comment:
There are also `crds/cilium/*.yaml` and `crds/internal/*.yaml`.

Reply:
No newline.

Reply:
this is an rbac for users, does it make sense to provide users with the ability to manage internal resources? We decided not to. If you think so, we can add the necessary resources